config_complete.toml

```diff
@@ -137,6 +94,57 @@ Logger = "logrus"
 # metrics to a Honeycomb dataset.
 Metrics = "honeycomb"
 
+#########################
+## Peer Management     ##
+#########################
+
+# [PeerManagement]
+# Type = "file"
+# Peers is the list of all servers participating in this proxy cluster. Events
+# will be sharded evenly across all peers based on the Trace ID. Values here
+# should be the base URL used to access the peer, and should include scheme,
+# hostname (or ip address) and port. All servers in the cluster should be in
+# this list, including this host.
+# This list is only used when the 'peer_type' command line flag is set to
```

Comment:
This comment and the one below both still reference the `peer-type` command line flag.

Reply:
🤦‍♂️ will update!